### DISCUSSION TITLE
First line as title, without "Title:"?

### DISCUSSION THREAD

**Discussion by swduncan** [getpelican/pelican#3414] [Q&A]
Is there any way to get Pelican to take the first line as the title, but without requiring "title:" (I use markdown)? Would it be possible to make a plugin do that?

OR

Is there a way to put the meta data for content at the bottom?

Thanks!

**Comment by mcepl** [comment]
Most likely a duplicate of https://github.com/getpelican/pelican/discussions/3290

**Reply by justinmayer** [reply]
Indeed, this appears to be a duplicate. Closing in favor of the existing discussion.
